DRAFT.md

```diff
@@ -752,7 +751,7 @@ explicit constexpr indirect(allocator_arg_t, const Allocator& a);
 7. _Mandates_: `T` is a complete type.
 
 8. _Effects_: `alloc` is direct-non-list-initialized with `a`. Value
```

Comment:
```suggestion
8. _Effects_: `alloc` is direct-non-list-initialized with `a`. 
```

Github won't let me make a suggestion that spans this line and the next green one, because of the red line between them, but I think this word should go as explained next...